### DISCUSSION TITLE
Execute a script on enablement

### DISCUSSION THREAD

**Discussion by ivailopankow** [Tampermonkey/tampermonkey#1752] [General]
Hy folks, I want to be able to run scripts when the page is already loaded and I enable a specific script. 
Example use-case: I have a site that allows me to enter various search criteria. I have two scripts saved and disabled for specific searches. Sometimes I want to. execute the first script/search and at other time I want to execute the second one. 
Is there a way for these scripts to be executed on activation?

Thanks

**Comment by derjanb** [comment]
I think

`// @run-at context-menu`

is what you're looking for. -> http://tmnk.net/docs#meta:run_at
